fields/default/fields_list.json

```diff
@@ -329,6 +329,16 @@
             "base.alt"
         ]
     },
+    {
+        "path": "tech_data.engine.standarts.emission.euro",
+        "description": "Характеристики ТС: Экологический евростандарт двигателя",
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "regactions.registry"
```

Comment:
Need to update `fillable_by` for other fields that may be filled by new source.

Reply:
Fixed for all fields filled by `regactions.registry`.